app/Notifications/GoogleDriveImportFailed.php

```
@@ -15,12 +15,12 @@ public function __construct(public GoogleDriveConfig $config, public string $err
     {
     }
 
-    public function via($notifiable)
+    public function via($notifiable): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)